client/js/store/modules/auth.js

```diff
@@ -59,7 +63,49 @@ export default {
     async requestSignup(context, { username, password }) {
       return handleLogin(context, username, password, true);
     },
-
+    async submitEmail(context, { username, email }) {
+      try {
+        const response = await fetch(`/user/submitEmail`, {
```

Comment:
pref single quotes over backticks unless you're using variables
```suggestion
        const response = await fetch('/user/submitEmail', {
```